postfix.yaml

```diff
@@ -207,7 +207,15 @@ subpackages:
   - range: _subpackages
     name: postfix-${{range.key}}
     description: "${{range.key}} map support for postfix"
+    dependencies:
+      runtime:
+        - mariadb-dev
     pipeline:
+      # test:
+      #   pipeline:
+      #     - uses: test/ldd-check
+      #       with:
+      #         packages: postfix-${{range.key}}:
```

Comment:
uncomment?

Reply:
The package currently fails to build from source so the test will fail until the package is rebuilt and the runtime dependency is picked up.